pkg/minikube/bootstrapper/images/images.go

```diff
@@ -145,7 +145,7 @@ func KindNet(repo string) string {
 	if repo == "" {
 		repo = "kindest"
 	}
-	return path.Join(repo, "kindnetd:v20210220-5b7e6d01")
+	return path.Join(repo, "kindnetd:v20210326-1e038dc5")
```

Comment:
Might be a good idea to add a comment to show where to find the latest tag (https://hub.docker.com/r/kindest/kindnetd/tags)? Could be useful for someone unfamiliar

Reply:
even better to point to kind source code, we wounld'nt wanna use any Latest Build that is not tesed and used by kind itself

Reply:
yep, it's from https://hub.docker.com/r/kindest/kindnetd/tags - i'll add it as a comment there
https://hub.docker.com/r/kindest/kindnetd says: "These images are from kind.sigs.k8s.io, prebuilt by the maintainer team."
i'll also add: "Source code is at: https://github.com/kubernetes-sigs/kind/tree/master/images/kindnetd"

Reply:
added above